source/products/roland/haus-und-grund/rest-api.rst

```diff
@@ -0,0 +1,217 @@
+REST API - Roland - Haus und Grund
+======================
+
+.. note::  Refer to :ref:`REST API page<rest_api>` for a more complete documentation regarding the necessary requests before performing and building your own.
+
+**Please remember to use the API specific item and touchpoint .**
```

Comment:
This file is usually built by BE developer. Please check with someone from product team - AJ or Reinis if it's fine.

Reply:
Do I actually need this at all? As far as I understand, policies will be bought only from webapp not through API? 🤔

Reply:
I think you can remove this file because this is done by BE if it's needed.

Reply:
Sneaky, want to be silent in BE team.

Reply:
What you mean by that?

Reply:
Joking, sorry. :(